daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Runner.scala

```diff
@@ -178,6 +178,12 @@ object ParticipantsJsonProtocol extends DefaultJsonProtocol {
 // Function that requires an argument.
 sealed abstract class Script extends Product with Serializable
 object Script {
+
+  // For now, we do not care of the logging context for Daml-Script, so we create a
+  // global dummy context, we can feed the Speedy Machine and the Scenario service with.
+  private[script] val DummyLoggingContext: LoggingContext =
+    LoggingContext.newLoggingContext(identity)
```

Comment:
The `LoggingContext` API  is designed to not that :cry:

Reply:
I’m not following, it seems like you got it working?

Reply:
It is working and ugly.